app/posts/[...filename]/client-page.tsx

```diff
@@ -0,0 +1,29 @@
+"use client"
+import { useTina } from "tinacms/dist/react";
+import { PostQuery } from "../../../tina/__generated__/types";
+interface ClientPageProps {
```

Comment:
**issue**: There should be vertical space after imports and after interface declarations - please investigate why ESLint is not detecting style issues.

Reply:
Added eslint rules